packages/fxa-settings/src/components/DataCollection/index.stories.tsx

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { DataCollection } from '.';
+
+storiesOf('Components/DataCollection', module).add('default', () => (
```

Comment:
I would like to write stories in the [CSF format](https://storybook.js.org/docs/react/api/csf) like we are for stories in the auth-server but I kept getting eslint complaints and wasn't sure what the diff was. I'll file a maintenance task to make sure we can use the new format in Settings and maybe convert one or two files for reference.